skt/runner.py

```diff
@@ -662,11 +662,6 @@ def run(self, url, max_aborted, release, wait=False,
                 self.cleanup_handler()
             ret = SKT_ERROR
 
-        if ret == SKT_ERROR:
-            return ret
-        if not wait:
-            return ret
-
```

Comment:
Ha-ha, this is a good one :smile: 
Only, this is rather a "NOOP" or "useless" code, but it does get executed, so it's not "dead".